src/praisonai-train/praisonai_train/cli/commands/train.py

```
@@ -30,24 +30,124 @@ def train_callback():
+    ("chat_template", "--chat-template", "Chat template name."),
+)
+
+
 @app.command("llm")
 def train_llm(
-    dataset: str = typer.Argument(..., help="Training dataset path"),
+    dataset: Optional[str] = typer.Argument(
+        None, help="Training dataset path. Optional when --config names one."),
+    config: Optional[Path] = typer.Option(
+        None, "--config", "-c", exists=True,
+        help="Training config YAML. Flags below override it."),
```

Ruff (0.16.2)

[warning] Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)